docs/modules/ROOT/pages/references/parameters.adoc

```diff
@@ -39,6 +39,23 @@ A dictionary holding the `.spec` for cluster logging.
 See the https://docs.openshift.com/container-platform/latest/logging/config/cluster-logging-configuring-cr.html[OpenShift docs] for available parameters.
 
 
+== `clusterLogForwarder`
+
+[horizontal]
+type:: dictionary
+default:: see `defaults.yml`
+
+A dictionary holding the `.spec.outputs`, `.spec.inputs` and `.spec.pipelines` for log forwarding.
+
```

Comment:
I'd like to see a couple sentences here to describe what the default config is doing. As far as I can tell (after reading the OCP docs) it just ensures that the logs are still forwarded to the in-cluster ES, but it would be nice to have this clearly documented here.

Reply:
True. Clarified configuration in documentation.